README.md

```diff
@@ -35,16 +32,21 @@ to listen via HTTPS, then don't use HTTP.
 
 ### Running via Docker
 
-Build the dockerfile and then run it:
+Prebuilt Docker images are published on [Docker Hub].
 
 ```sh
-$ docker build . -t http-proxy
-$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
+$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 twilightrs/http-proxy
+# Or with metrics enabled
+$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 twilightrs/http-proxy:metrics
 ```
 
 This will set the discord token to `"my token"` and map the bound port to port
 3000 on the host machine.
 
+Images come in multiple different variants for metrics, ARM, Discord API v8 and
+v6. You can use these with their corresponding image tags found on
+[Docker Hub][docker-hub-tags].
```

Comment:
Could you either rename this link to show that it goes to the tags page specially, or use the same link as Docker Hub above?